pkg/controllers/resourceconsist/consister.go

```diff
@@ -201,13 +202,60 @@ func (r *Consist) syncEmployees(ctx context.Context, employer client.Object, exp
 	toAddLifecycleFlzEmployees, toDeleteLifecycleFlzEmployees := r.getToAddDeleteLifecycleFlzEmployees(
```

Comment:
Should we use the `names` recorded on annotation as the `currentEmployees`? Not the `names` recorded in backend service.

Reply:
names used here is for add/remove finalizers for pods, names used in diff is employeeId(names) in backend service